framework/test/mocha/setup.js

```diff
@@ -56,6 +56,7 @@ if (process.env.LOG_DB_EVENTS === 'true') {
 
 testContext.config = config;
 testContext.config.constants = _.cloneDeep(app.constants);
+testContext.config.constants.DELEGATE_LIST_ROUND_OFFSET = 0;
```

Comment:
Hmm, maybe adding a comment for setting this constant here would be nice, even I didn't understand it for a second, can you imagine :P :D